pkg/endpoint/bpf.go

```diff
@@ -926,7 +927,7 @@ func (e *Endpoint) deletePolicyKey(keyToDelete policy.Key) error {
 	return nil
 }
 
-func (e *Endpoint) e.addPolicyKey(keyToAdd policy.Key, entry policy.MapStateEntry) error {
```

Comment:
Can you fix this in the `policy: Refactor policymap updates.` commit?

Reply:
Done!